addons/skin.pm3.hd/720p/Font.xml

```diff
@@ -1,181 +1,469 @@
-            <filename>unifont-9.0.06.ttf</filename>
-            <size>19</size>
-        </font>
-        <font>
-            <name>font12</name>
-            <filename>unifont-9.0.06.ttf</filename>
-            <size>20</size>
-        </font>
-        <font>
-            <name>font12b</name>
-            <filename>unifont-9.0.06.ttf</filename>
-            <size>20</size>
-            <aspect>.9</aspect>
-        </font>
-        <font>
-            <name>font13</name>
-            <filename>unifont-9.0.06.ttf</filename>
-            <size>23</size>
-        </font>
-        <font>
-            <name>font18</name>
-            <filename>unifont-9.0.06.ttf</filename>
-            <size>32</size>
-        </font>
-        <font>
-            <name>font18b</name>
-            <filename>unifont-9.0.06.ttf</filename>
-            <size>32</size>
-        </font>
-        <font>
-            <name>font20</name>
-            <filename>unifont-9.0.06.ttf</filename>
-            <size>25</size>
-        </font>
-        <font>
-            <name>font22</name>
-            <filename>unifont-9.0.06.ttf</filename>
-            <size>28</size>
-        </font>
-        <font>
-            <name>font25</name>
-            <filename>unifont-9.0.06.ttf</filename>
-            <size>35</size>
-        </font>
-        <font>
-            <name>font35b</name>
-            <filename>unifont-9.0.06.ttf</filename>
-            <size>80</size>
-            <aspect>.8</aspect>
-        </font>
-    </fontset>
-</fonts>
\ No newline at end of file
+	
+	<fontset id="Default" unicode="true">
+	 	<font>
+			<name>font10</name>
+			<filename>LiberationSans-Regular.ttf</filename>
+			<size>17</size>
+		</font>
+	 	<font>
+			<name>font11</name>
+			<filename>LiberationSans-Regular.ttf</filename>
+			<size>21</size>
+		</font>
+	 	<font>
+			<name>font12</name>
+			<filename>LiberationSans-Regular.ttf</filename>
+			<size>22</size>
+		</font>
+	 	<font>
+			<name>font12b</name>
+			<filename>LiberationSans-Regular.ttf</filename>
+			<size>22</size>
+			<aspect>.9</aspect>
+		</font>
+	 	<font>
+			<name>font13</name>
+			<filename>LiberationSans-Regular.ttf</filename>
+			<size>25</size>
+		</font>
+	 	<font>
+			<name>font18</name>
+			<filename>humanist_521_condensed_bt.ttf</filename>
+			<size>34</size>
+		</font>
+	 	<font>
+			<name>font18b</name>
+			<filename>humanist_521_bt.ttf</filename>
+			<size>34</size>
+		</font>
+	 	<font>
+			<name>font20</name>
+			<filename>LiberationSans-Regular.ttf</filename>
+			<size>27</size>
+		</font>
+	 	<font>
+			<name>font22</name>
+			<filename>LiberationSans-Regular.ttf</filename>
+			<size>30</size>
+		</font>
+	 	<font>
+			<name>font25</name>
+			<filename>LiberationSans-Regular.ttf</filename>
+			<size>37</size>
+		</font>
+	 	<font>
+			<name>font35b</name>
+			<filename>LiberationSans-Regular.ttf</filename>
+			<size>82</size>
+			<aspect>.8</aspect>
+		</font>
+
+<!-- Addon fonts -->
+
+<font>
```

Comment:
sorry, now i cannot say which version of Kodi pm3.hd i use to develop Font.xml - i think from 16. i`ve added couple <fontset> Arial group - no more.